app/controllers/api/v1/donor_conditions_controller.rb

```diff
@@ -16,12 +16,7 @@ class DonorConditionsController < Api::V1::ApiController
     api :GET, '/v1/donor_conditions', "List all donor conditions."
     param :ids, Array, of: Integer, desc: "Filter by donor condition ids e.g. ids = [1,2,3,4]"
     def index
```

Comment:
We have this same code repeating three times in district_controller, rejection_reasons and donor_condition controllers so i think its worth making a parent method. We can have a custom code incase we need to modify the above code for any controller. 

I have done all other changes. Do let me know if i need to make any changes.